package.json

```diff
@@ -1,20 +1,23 @@
 {
-  "name": "@openfeature/js-sdk",
-  "version": "1.0.0",
+  "name": "@openfeature/web-sdk",
+  "version": "1.0.99",
   "description": "OpenFeature SDK for JavaScript",
   "main": "./dist/cjs/index.js",
   "module": "./dist/esm/index.js",
-  "types": "./dist/types/index.d.ts",
+  "types": "./dist/types/client/index.d.ts",
```

Comment:
if we wanted to actually build this in this repo, we'd need to maintain 2 package.json files.